inst/pages/Introductory-workflow.qmd

```diff
@@ -0,0 +1,353 @@
+# Introductory
+
+<div id="workflow">
+
+<img src="images/logo.svg" alt="logo workflow" id="banner"/>
+
+# Introduction
+
+Hello and welcome to a complete workflow using the latest R/Bioconductor tools for microbiome data science. In this tutorial we’ll walk you through some basic steps of a composition analysis study using OMA. These will be applicable to almost any of your projects and will help you understand the fundamental concepts that will sky-rocket 🚀 your future microbiome analyses.
+
+# Importing data
+
+When using microbiome packages, there are many different ways to import your data. Let's first load the mia package:
+
+```{r}
+#| label: loadMia
+#| output: false
+library(mia)
+```
+
+```{r}
+#| label: loadDT
+#| echo: false 
+#| output: false
+library(DT)
+```
```

Comment:
Better to have one chunk that loads all the required packages
Similar to this
```

```{r setup}
# List of packages that we need
packages <- c(
    "ggplot2", "knitr", "mia", "dplyr", "miaViz", "MOFA2",  "patchwork", "reticulate",
    "MGnifyR", "reshape2", "GGally", "IntegratedLearner", "SuperLearner",
    "bartMachine", "mcmcplots", "tidyverse", "UpSetR", "shadowtext", "ComplexHeatmap",
    "stringr", "cowplot", "caret", "scater", "ggsignif"
    )

# Get packages that are already installed installed
packages_already_installed <- packages[ packages %in% installed.packages() ]

# Get packages that need to be installed
packages_need_to_install <- setdiff( packages, packages_already_installed )

# Loads BiocManager into the session. Install it if it not already installed.
if( !require("BiocManager") ){
    install.packages("BiocManager")
    library("BiocManager")
}

# If there are packages that need to be installed, installs them with BiocManager
# Updates old packages.
if( length(packages_need_to_install) > 0 ) {
   install(packages_need_to_install, ask = FALSE)
}

# Load all packages into session. Stop if there are packages that were not
# successfully loaded
pkgs_not_loaded <- !sapply(packages, require, character.only = TRUE)
pkgs_not_loaded <- names(pkgs_not_loaded)[ pkgs_not_loaded ]
if( length(pkgs_not_loaded) > 0 ){
    stop("Error in loading the following packages into the session: '", paste0(pkgs_not_loaded, collapse = "', '"), "'")
}
```
```